src/app/components/Footer/index.tsx

```diff
@@ -63,6 +64,15 @@ export function Footer() {
             </p>
           </div>
         </div>
+        <div className="text-center small text-lightGrey">
+          <a
+            href={`https://github.com/DistributedCollective/Sovryn-frontend/commit/${commitHash}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {commitHash}
+          </a>
+        </div>
```

Comment:
Maybe add condition so it wouldn't show if there is no commit id? `{commitHash && <div>...</div>}`

Reply:
Good idea, no need to render the container elements if the commit id isn't present